core/services/functions/listener_test.go

```diff
@@ -94,6 +99,7 @@ func NewFunctionsListenerUniverse(t *testing.T, timeoutSec int, pruneFrequencySe
 	}
 	eaClient := functions_mocks.NewExternalAdapterClient(t)
 	bridgeAccessor := functions_mocks.NewBridgeAccessor(t)
+	decryptor := threshold_mocks.NewDecryptor(t)
```

Comment:
How come this doesn't break the test named "HandleOracleRequestCBORParsingCorrect"? That test has non-empty secrets so it should make calls FetchEncryptedSecrets and Decrypt...

I'm confused.

Reply:
This test is missing the SecretsLocation param and therefor fails the check `requestData.SecretsLocation == LocationRemote`